src/controllers/userController.ts

```diff
@@ -86,3 +87,33 @@ export const deleteUser = async (
     return next(error)
   }
 }
+
+export const toggleUserActiveStatus = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  try {
+    const userId = req.params.userId
+    const { active } = req.body
+
+    if (typeof active !== "boolean") {
+      return res.status(400).json({
+        message: "Active status must be a boolean",
+      })
+    }
```

Comment:
I'd suggest to use joi for validation here, since we're using it in other routesl like `updateUser` and adjust the updateUserSchema accordingly.

```suggestion
   await toggleUserActiveStatusSchema.validateAsync({ active })
```